src/main/webapp/app/core/user/settings/learner-profile/course-learner-profile.component.html

```diff
@@ -0,0 +1,63 @@
+<div class="d-flex justify-content-between">
+    <h2 jhiTranslate="artemisApp.learnerProfile.courseLearnerProfile.title"></h2>
+    @if (editing) {
+        <button type="button" class="btn btn-primary" id="apply-changes-button" (click)="update()">
+            <fa-icon class="ng" [icon]="faSave" />
+            {{ 'artemisApp.userSettings.saveChanges' | artemisTranslate }}
+        </button>
+    }
+</div>
+<hr />
+<div class="row align-items-center">
+    <div class="col">
+        <span jhiTranslate="artemisApp.learnerProfile.courseLearnerProfile.description"></span><br />
+        <div class="d-flex align-items-center my-2">
+            <span class="dot background-color ms-1 me-2"></span>
+            <span jhiTranslate="artemisApp.learnerProfile.courseLearnerProfile.initialState"></span>
+            <jhi-help-icon class="ps-1" text="artemisApp.learnerProfile.initialValueHint" />
+            <span class="dot bg-primary ms-3 me-2"></span>
+            <span jhiTranslate="artemisApp.learnerProfile.courseLearnerProfile.currentState"></span>
+            <jhi-help-icon class="ps-1" text="artemisApp.learnerProfile.currentValueHint" />
+        </div>
+    </div>
+    <select class="form-select col-auto w-auto" (change)="courseChanged($event)">
+        <option value="-1" jhiTranslate="artemisApp.learnerProfile.courseLearnerProfile.selectCourse" selected></option>
+        @for (course of courses; track course.id) {
+            <option value="{{ course.id }}">{{ course.title }}</option>
+        }
+    </select>
+</div>
+<div [ngClass]="{ 'd-none': disabled }">
+    <div class="row row-cols-2 gx-5 align-items-end">
+        <div class="col">
+            <jhi-double-slider
```

Comment:
I do not understand the naming behind this, why is it called `double-slider`? 

Not required to be changed for my approve

Reply:
This represtents two range inputs therefore double slider. I tried to design the code, so it can be adapted to other use cases.